linera-core/src/local_node.rs

```diff
@@ -142,8 +151,15 @@ where
 
 impl<S> LocalNodeClient<S> {
     pub fn new(state: WorkerState<S>, notifier: Arc<Notifier<Notification>>) -> Self {
+        use rand::SeedableRng as _;
+
+        // The following seed is chosen to have equal numbers of 1s and 0s, as advised by
+        // https://docs.rs/rand/latest/rand/rngs/struct.SmallRng.html
+        // Specifically, it's "01" × 32 in binary
+        const RNG_SEED: u64 = 6148914691236517205;
```

Comment:
While not critical, this is not amazing, and we should probably improve this after testnet -> create a TODO and task?